utils/spacewalk-common-channels.ini

```diff
@@ -1221,6 +1281,26 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/SLE15-Uyuni-Client-Tools/SLE_15/
 
+[sles15-sap-sp2-uyuni-client]
+name     = Uyuni Client Tools for SLES15 SP2 %(arch)s
+archs    =  x86_64, s390x, aarch64, ppc64le
+base_channels = sle-product-sles_sap15-sp2-pool-%(arch)s
+checksum = sha256
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/SLE15-Uyuni-Client-Tools/SLE_15/repodata/repomd.xml.key
```

Comment:
I guess the URL should be `https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/SLE15-Uyuni-Client-Tools/SLE_15/repodata/repomd.xml.key` here (Stable tools)...

Reply:
I copied this url block from the other SLES15 SP2 parts above. Shall I just edit this from Master to Stable anyway?
See for e.g. line: 1249

Reply:
Seems you unearthed a bug. Yes, please, change this line to Stable, and same for 1292. I will create a PR to fix SLE15SP2 above.

Reply:
changed from master to stable in dd89b5f

Reply:
The PR to fix the other repositories is at https://github.com/uyuni-project/uyuni/pull/3636